sdk/identity/identity/test/internal/node/interactiveBrowserCredential.spec.ts

```diff
@@ -29,8 +39,8 @@ describe("InteractiveBrowserCredential (internal)", function() {
 
   it("Throws an expected error if no browser is available", async function(this: Context) {
     // On Node 8, URL is not defined. We use URL on the msalOpenBrowser.ts file.
-    if (isNode8) {
-      this.skip();
+    if (process.version.startsWith("v8.")) {
+      global.URL = URL;
```

Comment:
Why are the changes to this file necessary?

Reply:
The definition of `isNode8` was provided by the tokenCache modules, so I figured it would be simplest to just fix this one test than to keep it around in the runtime code.

Reply:
This test doesn't make sense in the browser since it stubs "open" to throw an error, to emulate when `open` is unable to find a browser. It's not clear to me how this function passes in the browser at all, unless the bundling of this credential is incorrect (remember there are two versions of this credential, one of them is specifically for the browsers).

Reply:
Alright, thank you!